integration-cli/docker_cli_by_digest_test.go

```diff
@@ -341,6 +341,13 @@ func (s *DockerRegistrySuite) TestListDanglingImagesWithDigests(c *testing.T) {
 	// make sure repo shown, tag=<none>, digest = $digest1
 	re1 := regexp.MustCompile(`\s*` + repoName + `\s*<none>\s*` + digest1.String() + `\s`)
```

Comment:
Is this case already covered by TestListImagesWithDigests above?

If it is, it may be cleaner to `t.Skip()` the whole test

Reply:
Maybe this one test is actually multiple sub-tests 🤔 we could make sub-tests and skip those, but I guess we'd have to (temporarily) disable the "integration-cli is deprecated" check.